content/en/docs/Performance/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Performance"
+weight: 150
+date: 2020-11-20
+description: >
+  Baker performance test cases
+---
+
+We report in this page some practical examples of Baker performances we could measure in the
+NextRoll production environment.
+
+Within NextRoll, Baker is often executed on AWS EC2 instances, and thus you find in this page
+many references to
+[EC2 instance types](https://aws.amazon.com/ec2/instance-types/) (`c5.2xlarge`, `c5.2xlarge`, etc.).
+
+### Read from S3 and write to local disk
+
+On an AWS EC2 instance of size `c5.2xlarge`, Baker can read zstandard records from S3, uncompress
+them and apply a basic filtering logic, compressing them back on local files with zstandard at
+compression level 3 and long range mode at 27 using ~90% of capacity of each vCPU (8 in total) and
+~3.5GB of RAM.  
+
+It reads and writes a total of 94 million records in 8'51" (~178k r/w records per second).
+
+On a `c5.2xlarge` instance (48vCPUs) the same test takes 2'2" (~775k r/w records per second).
+
+For this test we use 711 zstd compressed files for a total of 17 GB of compressed size and 374 GB
+of uncompressed size. The average size of each record is ~4.5 KB.
+
+### Read from S3 and write to DynamoDB (in the same region)
+
+On a `c5.4xlarge` instance, Baker reads zstd compressed files from S3 writing to DynamoDB (configured
+with 20k write capacity units) at an average speed of 60k records/s (the average size of each record
+is 4.3 KB) using less than 1 GB of memory and ~300% of the total CPU capacity (less than 20% for
```

Comment:
This suggestion just removes `~`
```suggestion
is 4.3 KB) using less than 1 GB of memory and 300% of the total CPU capacity (less than 20% for
```

Not sure what _less than 20%_ means? _less than 20% of CPU usage_ maybe?